interface/modules/zend_modules/public/js/multipledb/jquery.validate.min.js

```diff
@@ -0,0 +1,4 @@
+/*! jQuery Validation Plugin - v1.15.0 - 2/24/2016

```

Comment:
This library is already in public/assets, so use this instead:
public/assets/jquery-validation-1-13-0/dist/jquery.validate.min.js
(if 1.15 is required, then I can add that)